public-api/src/main/java/org/pmiops/workbench/cdr/dao/AchillesAnalysisDao.java

```diff
@@ -18,6 +18,16 @@
     )
     AchillesAnalysis findConceptAnalysisResults(String concept_id,Long analysisId);
 
+    @Query(value = "select a from AchillesAnalysis a where a.analysisId=?1")
+    AchillesAnalysis findAnalysisById(Long analysisId);
+
+    @Query(value = "select distinct a from AchillesAnalysis a left join FETCH a.results as r " +
```

Comment:
Please generalize this. It works for any concept id potentially. Not that we have teh data for all yet. But just rename.